src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -498,3 +498,22 @@ def labels(self) -> List[str]:
     @handle_orchestrator_error('host')
     def inventory(self, refresh=None):
         return get_inventories(None, refresh)
+
+    @Endpoint('GET')
+    @ReadPermission
+    @handle_orchestrator_error('host')
+    def hosts_per_service(self) -> List:
```

Comment:
The endpoint name is a little bit weird: essentially you're retrieving a service list (orch.services.list()) and adding host count to the payload, so what about renaming to `services` so the endpoint URL is:
`/ui-api/host/services`. IMHO is more simple and self-explanatory.